README.md

```diff
@@ -14,3 +15,33 @@ In addition, we are trying to have a well-designed structure ourselves so that v
 
 Through these activities, ONE-vscode can provide a differentiation that cannot be experienced only with the existing CLI by linking the toolchain with the execution environment, ONE Runtime, or a target simulator corresponding to the backend compiler. This will eventually lead to fun and high productivity for developers.
 
+## Overview and tutorials
+
+- [ONE-vscode extension overview](./docs/Overview.md)
+- [How to install?](./docs/HowToInstall.md)
+- [How to use?](./docs/HowToUse.md)
+- [Glossary](./docs/Glossary.md)
+
+## Features
+
+<img src="https://user-images.githubusercontent.com/7223627/175224054-0df3109a-f4ef-4a12-a7be-ab50f01b77af.png" width=500 />
```

Comment:
Could you make this image a bit larger so that user can read letters inside the image?

Reply:
How about the following?
- `ONE View Explorer` -> `ONE Explorer View`
- `Toolchain View Explorer` -> `Toolchain View`
- `Device View Explorer` -> `Target Device View`

https://github.com/Samsung/ONE-vscode/blob/e7536898c785974753e029e40248b4da3cf91f94/package.json#L31-L39

/cc @dayo09, @jyoungyun, @struss

Reply:
It's easy to change the size of the image. Q. Will I change the size of the others? or do you want to change the only one?

Reply:
I think that any image sizes with which letters in the image can be shown well to users will be fine. :-)

Reply:
I'll try with size=600. Let me get your feedback.